### PR TITLE
Fixes #130: Create skills from Loklak

### LINKED ISSUE #130: Create skills from loklak API
**Actual Behaviour**

Currently, there are no skills from the [Loklak API](http://api.loklak.org)

**Expected Behaviour**

One can use the following URL to produce responses
```
http://api.loklak.org/api/search.json?timezoneOffset=-330&q=fossasia
```
**Sample Queries - Tweets about * | Trending tweets about***
A table action type can be used to generate tweets in a table for the path - 
``` $.statuses```

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #130 

Changes: 
- Added queries to get tweets about a query

Dreampad Link : http://dream.susi.ai/p/loklak

Screenshots for the change: 
screenshot from 2017-06-14 08 03 16

### CODE REVIEW

**Review by Manveer695 — APPROVED** [review]

File: models/general/knowledge/en/loklak.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Return a list of universities in a country
+tweets about *| tweet *| 5 tweets about *|Latest tweets about *
```
Comment (AliAyub007) [inline models/general/knowledge/en/loklak.txt]: I suggest you should add more questions to it to make this skill responsive like
Give me tweets about * 
Tell me tweets about *
what are latest tweets about * 
what are the latest tweets about * 
Can you give me tweets about * 
I want tweets about * 
I need tweets about * 

and similarly more like this.

**Comment by rishiraj824** [comment]
@AliAyub007 Please review

**Review by AliAyub007 — APPROVED** [review]
Kindly squash the commits 👍

**Comment by rishiraj824** [comment]
@AliAyub007 Done

**Review by AliAyub007 — APPROVED** [review]

**Review by hpdang — CHANGES_REQUESTED** [review]
@rishiraj824 good start. But please consider how to name your skill. Does this skill provide any information about loklak? loklak is just another API like wolframalpha. Your current questions are fine but please think of more things you can do with tweets. Can you tell people's favorite drama this month/week from the tweets? can you tell most view news articles? anything about celebrities? and can we find a twitter handle of a person based on his/her full name, or name + company?  Please make it your focus of Milestone 7. Thanks.

**Comment by rishiraj824** [comment]
@hpdang Sure. I'll try and see how we can get all this data as a response.

**Comment by rishiraj824** [comment]
@hpdang Updated the PR with some more options
 - One can find tweets from particular users
screenshot from 2017-06-20 10 37 50
-  One can find tweets having certain user mentions
screenshot from 2017-06-20 10 39 31

**Review by dynamitechetan — APPROVED** [review]

**Comment by hpdang** [comment]
@rishiraj824 thank you. Can you still think of more things we can do with Loklak data please!

**Pull request merged**